frontend_app_kanban/src/css/index.styl

```diff
@@ -235,9 +235,13 @@ cardWidth = 280px
             margin standardSpacing 0
 
 @media (max-width: max-xs)
+  body:not(.tox-fullscreen) .kanban__KanbanPopup
+    transform translate(-50%, 0%) !important
```

Comment:
is !important really necessary? it doesn't seem like it is overriding a library rule

Reply:
It is overriding the default behavior of the cardPopup, which has top priority overs this class, meaning the `!important` is required

Reply:
why is the issue only for .kanban__KanbanPopup?
shouldn't it be the same issue for every popup on mobile?

Reply:
Because the kanban popup is bigger than the other popups and thus needs special treatment

Reply:
From what I've tried, the only way I could apply this property was to write in the cardPopup style file, so I guess this time !important is appropriate ?